app/server/server_test.go

```diff
@@ -0,0 +1,44 @@
+package server
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"time"
+
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ProcessStats(t *testing.T) {
+	srv := NewRLBStatsServer("v1", 0)
+	ts := httptest.NewServer(srv.routes())
+	defer ts.Close()
+	defer srv.Shutdown()
+
+	reqrec := LogRecord{
+		FromIP:   "127.0.0.1",
+		Fname:    "test.mp3",
+		DestHost: "127.0.0.2",
+	}
+
+	data, err := json.Marshal(&reqrec)
+	assert.Nil(t, err)
+
+	client := &http.Client{Timeout: 5 * time.Second}
+
+	resp, err := client.Post(ts.URL+"/stats", "application/json", bytes.NewReader([]byte{}))
+	assert.Nil(t, err)
```

Comment:
for such unclear use cases (i have no clue why empty body is ok here) is better to use third param in assert with message or just add a comment. Smth like assert.NoError(t, err, "empty body allowed for ...") And making error check with NoError generally preferebale as it will pring nice details about an unexepcted error.